p/vmt/products.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+module.exports = [ // todo: what is `512`?
+	{
+		id: 'long-distance-train',
+		mode: 'train',
+		bitmasks: [1, 2, 4],
+		name: 'long-distance train',
```

Comment:
Stupid little question: Do we capitalize the names here (as in INVG) or not? We should be consistent.

Reply:
I don't know. These descriptions are not internationalised anyways, so I rather see them as useful for prototype-level apps and as a fallback.